manifests/params.pp

```diff
@@ -20,6 +20,26 @@
     'Debian' => 'apt',
     default  => 'archive'
   }
+
+  if $facts['os']['name'] == 'Ubuntu' {
```

Comment:
Note that this same split is applicable for Debian from the next release:
Stable + Oldstable Python 2 https://packages.debian.org/buster/duplicity
Testing python 3 https://packages.debian.org/bullseye/duplicity
I'm not sure when the next release of this role will be but this might be relevant as the next Debian release is expected in 2-3 months.